### ISSUE TITLE
Have a clear button on text/search fields

### ISSUE THREAD

**Issue by iqualfragile** [gdotdesign/elm-ui#32]
at least optionally, maybe depending on user os.
its really comfortable to have those if you are using an os where you can select-and-middle-mouse-button to copy and paste.

**Comment by gdotdesign** [comment]
It has been added thanks to (**eimfach**) and will be available in the next release.

Usage:
```elm
input = Ui.Input.init "value" "placeholder"
inputWithClear = { input | showClearIcon = True }
```

**Comment by gdotdesign** [comment]
Available in `0.4.3`

**Comment by iqualfragile** [comment]
thanks for the work @eimfach & @gdotdesign
